_posts/2019-11-20-agile-reporting.md

```diff
@@ -0,0 +1,254 @@
+
+Let's take an example. 
+In our backlog, we have 3 features waiting, a total of 150 story points. 
+Now, you have to estimate how many sprints it would take to finish and by what date.
+
+Assume we start 01-01-2019 and work with sprints of 2 weeks. 
+The past 5 velocities were 60, 55, 40, 45, 51, averaging in 50. 3 sprints, great!  
+Estimated end date = 01-01-2019 + 6 weeks = 12-02-2019
+Now, let's put that in the tool.
+
+<img alt="Throughput Tool Input" src="{{ '/img/2019-11-20-agile-reporting/reporting-throughput-input.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+The result:
+
+<img alt="Throughput Tool Result" src="{{ '/img/2019-11-20-agile-reporting/reporting-throughput-results.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+So there's a 55% probability that the features will be done in 3 sprints.  
+However, there's a 100% probability that it will be done in 4 sprints. And that's just for 3 features worth of 150 story points.
+
+# Metrics
+
+During my role as team lead, I've also had to measure the performance and happiness of the team. 
+To do so, it helps to have raw data to analyse. 
+
+## Velocity
+
+The tool we use for managing our scrum process is Jira. 
+It contains various reports that can be generated based on your agile board.  
+But mostly,  I used the export to spreadsheet functionality.
+This provided me with a spreadsheet file with all the raw data of the entire backlog in Jira.
+
+With that spreadsheet, it's just a matter of using the various functions to analyse the data. 
+This gave me results such as the image below.
+
+<img alt="Raw Velocity" src="{{ '/img/2019-11-20-agile-reporting/reporting-velocity-raw.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+I also asked all team members to enter their hours per sprint in another spreadsheet. 
+Which resulted in the following:
+
+<img alt="Hours Per Sprint" src="{{ '/img/2019-11-20-agile-reporting/reporting-hours-input.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+All that data can be then be combined into a chart.
+
+<img alt="Velocity Chart" src="{{ '/img/2019-11-20-agile-reporting/reporting-velocities-chart.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+As always with velocity, it tells us what happened in the past. 
+But what does the chart above tell us?
+* The story points done per sprint are rising.
+* The last 2 sprints, the team did a very good estimation as they reached or surpassed their forecasted story points.
+* The relative velocity (what the team could have done if everyone was present) is always much higher
+** This could for example indicate that team members are often absent, which explains the difference between the velocity and the relative velocity.
```

Comment:
Two spaces to indent a bullet point.
```suggestion
  * This could for example indicate that team members are often absent, which explains the difference between the velocity and the relative velocity.
```